app/assets/stylesheets/application.scss

```diff
@@ -244,6 +244,9 @@ select {
   margin-bottom: 0px;
   font-weight: 300;
 }
+.title-link:hover {
```

Comment:
"go back" links in course page
course title takes you to courses/[id]/professors
professor title takes you to professor page

I think @keenadams wants to implement breadcrumbs at some point but for now,
this should make navigation easier